### ISSUE TITLE
GU0018 don't warn about public field

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Analyzers#281]
```cs
public static readonly Mock<IPlcClient> PlcClientMock = CreatePlcClientMock();
```
